docs/casestudies/unionbigdata.md

```diff
@@ -1,95 +1,89 @@
 
-![I3Plat平台架构图](./static/unionbigdata_02.png)
+![I3Plat Platform Architecture](./static/unionbigdata_02.png)
 
-项目目标是实现现地 ADC 上线、资源共享和云边端标准化，以减轻运维负荷、提升标准。I3Plat 旨在通过规范化和标准化 BOE 集团的 ADC 系统，为后续 ADC 建设提供样本和模板，降低成本和周期，提高生
-产和质检效率以及产品良率。包含系统管理员、资源配置员等用户角色，并涉及 ADC 推理、模型训练、数据共享等信息流，以及云端协同功能，确保 ADC 的自动缺陷分类生产过程，并提高模型和缺陷图片的
-利用率。
+The project aims to launch ADC at the factory sites, facilitate resource sharing, and achieve cloud-edge standardization to reduce the operational burden and uphold high standards. I3Plat is designed to streamline and standardize BOE Group's ADC systems across factories, providing a blueprint and reference for future ADC deployments. This aims to reduce costs and timeframes, and boost both production efficiency and the quality inspection process, ultimately improving product yields. It includes roles like system administrators, resource allocators and entails workflows for ADC inference, model training, data sharing, as well as cloud collaboration features, ensuring an automated defect classification process, and enhancing the usage of models and defective image data.
 
-## 产品与技术实现
+## 2. Product and Technical Implementation
 
-### 一、集群管理
+### 2.1. Cluster Management
 
-不同现地可将对应的 K8s 集群注册至中心云系统，中心云系统对多个现地的集群进行管理。
+Factories can register their respective K8s clusters with the central cloud system, which then manages these clusters from a single point.
 
-![集群管理](./static/unionbigdata_03.png)
+![Cluster Management](./static/unionbigdata_03.png)
 
-我们选择了 PULL 模式。
+We have chosen the PULL mode.
 
-为了降低 OP 的操作成本，我们在中心云提供了 step-by-step 的注册流程。
+To reduce the operational cost for operators, we offer a step-by-step registration process within the central cloud.
 
-1. 引导安装 karmada-agent。
-2. 使用 karmadactl token create 控制面生成 token。
-3. 引导注册 karmadactl register 。
-4. 在成员集群中编辑由 karmadactl register 创建的 deploy/karmada-agent 以确保其可以访问该成员集群的 kube-apiserver。
+1. Guide the installation of karmada-agent.
+2. Generate a token using karmadactl token create in the control plane.
+3. Proceed with the registration using karmadactl register.
+4. Edit the deploy/karmada-agent created by karmadactl register in the member cluster to ensure its access to the kube-apiserver of that member cluster.
 
-### 二、使用聚合层 API
+### 2.2. Using Aggregate Layer API
 
-通过 karmada-aggregator 组件提供的集群统一访问能力，我们可以在中心云实现可视化大屏等需要聚合成员集群的数据的功能。
+Through the cluster unification access provided by the karmada-aggregator component, we are able to implement functions in the central cloud that amalgamate data from member clusters, such as visualized dashboards.
 
-通常我们用 Service 来暴露 Java 实现的功能，并用 Java Fabric8 等客户端调用 kubectl get --raw 来实现调用：
+Typically we expose functions implemented in Java using Service, and invoke kubectl get --raw with Java Fabric8 or similar clients:
 
 ```
 /apis/cluster.karmada.io/v1alpha1/clusters/%s/proxy/api/v1/namespaces/%s/services/%s/proxy/%s
 ```
 
-#### 1、集群监控
+#### 2.2.1. Cluster Monitoring
```

Comment:
```suggestion
#### Cluster Monitoring
```
ditto. It need to change more than your expected if you add or remove sections later

Reply:
Got it, I've remove all the numbers. Should I add a new commit or squash them into one?

Reply:
Seems all right. It has only one commit now.

Reply:
> Seems all right. It has only one commit now.

Thanks for reviewing.